.github/workflows/ci.yaml

```diff
@@ -51,3 +52,7 @@ jobs:
                   XDG_CACHE_HOME: ~/.cache/bazel-repo
               working-directory: ${{ matrix.folder }}
               run: bazel --bazelrc=$GITHUB_WORKSPACE/.github/workflows/ci.bazelrc --bazelrc=.bazelrc test //...
+            - name: ./test.sh (e2e/worker)
```

Comment:
hmm; can you make this generic so it runs test.sh if it exists in the directory instead of making it specific to e2e/worker?

Reply:
Yes. I'll do that